### ISSUE TITLE
download auto-tag for inspection

### ISSUE THREAD

**Issue by nskitch** [GorillaStack/auto-tag#22]
I have some security concerns with hosting a 3rd party bucket and giving it permissions to execute in my secure AWS environment.  I'd like to host the autotag.zip in our own bucket so we can control when this code is updated.

I'm trying to download the existing auto-tag.zip file with no luck

**Comment by nskitch** [comment]
nm, figured out a method:

wget http://gorillastack-autotag-releases-ap-northeast-1.s3.amazonaws.com/autotag-0.3.0.zip

**Comment by nskitch** [comment]
Actually pretty surprised to see all the diffs between the projects /src folder and the hosted auto-tag.zip file.  The /src files this project need to be updated.

It's a little concerning to see code inserted to email the author of the project, not that I have a problem with this particular code, more that it's not in the repo.  I've not yet found anything that actually modifies the ingress rules or when this code is called.

     ```
_createClass(SecurityGroupSnitch, [{
    key: 'tagResource',
    value: function tagResource() {
      var _this = this;

      return new Promise(function (resolve, reject) {
        try {
          _this.mandrill('/messages/send', {
            message: {
              to: [{ email: 'elliott@gorillastack.com', name: 'Elliott Spira' }],
              from_email: 'hello@gorillastack.com',
              subject: 'Security Group Updated',
              text: 'Ingress rules changed - sg: ' + _this.event.requestParameters.groupId + ' by: ' + _this.event.userIdentity.arn
            }
          },
```

**Comment by em0ney** [comment]
Hi there @nskitch,

Interesting!  This should not be in the codebase.  I believe I authored that as a demo of how the repository could be forked for different use cases.  This use case was a hook to notify users off the back of a change in security groups.  I will review this over the weekend.  We must have accidentally pushed out a new zip file after some changes and that demo code would have slipped in.

I definitely have no intention to be reading security group changes.  Also worth pointing out that without a valid api key for mandrill, this would not run.

I'll check back in with you once I have cleaned up the project code vs hosted code issue.

Thanks!
Elliott
